Tests/XCUITests/HistoryTests.swift

```diff
@@ -38,45 +39,39 @@ class HistoryTests: BaseTestCase {
     }
 
     func testEmptyHistoryListFirstTime() {
-        navigator.performAction(Action.CloseURLBarOpen)
-        navigator.nowAt(NewTabScreen)
         // Go to History List from Top Sites and check it is empty
         navigator.goto(LibraryPanel_History)
         waitForExistence(app.tables.cells[HistoryPanelA11y.recentlyClosedCell])
-        XCTAssertTrue(app.tables.cells[HistoryPanelA11y.recentlyClosedCell].exists)
+        XCTAssertTrue(app.tables.cells[HistoryPanelA11y.recentlyClosedCell].staticTexts["Recently Closed"].exists)
+        XCTAssertTrue(app.tables[HistoryPanelA11y.tableView].staticTexts[emptyRecentlyClosedMesg].exists)
     }
 
     func testOpenSyncDevices() {
-        navigator.performAction(Action.CloseURLBarOpen)
-        navigator.nowAt(NewTabScreen)
+        // Firefox sync page should be available
         navigator.goto(TabTray)
         navigator.performAction(Action.ToggleSyncMode)
         waitForExistence(app.tables.cells.staticTexts["Firefox Sync"])
-        XCTAssertTrue(app.tables.buttons["Sign in to Sync"].exists, "Sing in button does not appear")
+        XCTAssertTrue(app.tables.buttons["Sync and Save Data"].exists, "Sign in button does not appear")
     }
 
     func testClearHistoryFromSettings() {
-        navigator.performAction(Action.CloseURLBarOpen)
-        navigator.nowAt(NewTabScreen)
         // Browse to have an item in history list
         navigator.goto(LibraryPanel_History)
-        waitForExistence(app.tables.cells[HistoryPanelA11y.recentlyClosedCell], timeout: 5)
```

Comment:
It looks like the test does not need the `timeout` values for `waitForExistence` commands.

Reply:
Maybe not in this case, but it is needed in others where we need to be sure the element is there to continue with the next action. Sometimes it takes than less five seconds, some others more... that's the value of that timeout.

Reply:
:) I've added two defaults: `TIMEOUT` and `TIMEOUT_LONG`. The values are `15` and `45` respectively. Let me be a bit generous in the shorter waits.